native/packages/react-native-bpk-component-button-link/readme.md

```diff
@@ -35,10 +35,8 @@ export default class App extends Component {
           title="Book flight"
           onPress={() => {}} />
         <BpkButtonLink
-          title="Book flight"
-          onPress={() => {}} />
-        <BpkButtonLink
-          title="Book flight"
```

Comment:
There were three buttons in the example code that were all the same, so I've removed one and made another showcase the `disabled` prop.